catalog/core/catalog-core-standardframework/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -364,6 +364,8 @@
     <reference id="defaultAttributeValueRegistry"
                interface="ddf.catalog.data.DefaultAttributeValueRegistry"/>
```

Comment:
The DefaultAttributeValueRegistry should be removed and replaced with a reference list of DefaultAttributeValues similar to what we are doing with AttributeInjectors in this PR.